.github/workflows/lint.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: Test
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  lint:
+    name: Pylint
```

Comment:
I decided to go with pylint instead of flake8 because it has some useful suggestions/warnings that flake8 doesn't have, but if we want to we can always run both.

Reply:
Works for me.